webdav/docs/static/styles.css

```diff
@@ -0,0 +1,338 @@
+body,
+html {
+	cursor: default;
+}
+
+body,
+div,
+dl,
+dt,
+dd,
+ul,
+ol,
+li,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+pre,
+form,
+fieldset,
+input,
+textarea,
+p,
+blockquote,
+th,
+td {
+	margin: 0;
+	padding: 0;
+}
+
+* {
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
+}
+
+:before,
+:after {
+	-webkit-box-sizing: border-box;
+	-moz-box-sizing: border-box;
+	box-sizing: border-box;
+}
+
+img,
+object,
+embed {
+	max-width: 100%;
+	height: auto;
+}
+
+object,
+embed {
+	height: 100%;
+}
+
+img {
+	margin: 1.25% 0;
+	-ms-interpolation-mode: bicubic;
+}
+
+html {
+	background-color: #F0F1F3;
+	padding: 2%;
+}
+
+body {
+	font-size: 16px;
+	line-height: 1.6;
+	font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol";
+	color: #242424;
+	max-width: 800px;
+	margin: 5% auto;
+}
+
+body::after {
+	clear: both;
+	content: "";
+	display: table;
+}
+
+header {
+	margin-bottom: 8%;
+}
+
+footer {
+	text-align: center;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5,
+h2 a {
+	color: #263A48;
+	font-weight: 500;
+	text-decoration: none;
+}
+
+h1,
+h2 {
+	font-size: 36px;
+	padding-bottom: 0.3em;
+	margin-bottom: 0.4em;
+	border-bottom: 1px solid #eee
+}
+
+h2 {
+	font-size: 22px;
+	padding-bottom: 0.6em;
+	margin-bottom: 0.6em;
+	margin-top: 2.5em;
+}
+
+h3 {
+	font-size: 18px;
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces (indentation)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6202733)